app/src/main/java/org/fossasia/openevent/app/login/LoginActivity.java

```diff
@@ -102,11 +84,13 @@ protected void onStart() {
             }
         });
 
-        btnLogin.setOnClickListener(v -> {
+        binding.loginContent.btnLogin.setOnClickListener(view -> {
+            if (!validator.validate())
```

Comment:
Could not able to log in with `https://....com/api/v1/` since validator doesn't validate this pattern. You may just change the default base URLs to `https://....com` or validate `https://....com/api/v1/`